client/state/utils.js

```diff
@@ -96,7 +117,16 @@ export const keyedReducer = ( keyName, reducer ) => {
 		);
 	}
 
+	const initialState = reducer( undefined, { type: '@@calypso/INIT' } );
```

Comment:
@samouri can you comment on whether you think this is sufficient here? I wasn't sure why it was being recreated on every dispatch

Reply:
seems fine to me